cloudera/tests/test_unit.py

```diff
@@ -236,12 +169,14 @@ def test_given_cloudera_check_when_bad_health_cluster_then_emits_cluster_health_
         )
 
 
-def test_given_cloudera_check_when_good_health_cluster_then_emits_cluster_health_ok(
```

Comment:
To reviewer: I removed this test since `test_given_cloudera_check_when_good_health_cluster_then_emits_cluster_metrics` was basically a superset of what this test was originally asserting.